pkg/codegen/go/gen.go

```diff
@@ -1388,7 +1389,7 @@ func (pkg *pkgContext) provideDefaultsFuncName(typ schema.Type) string {
 }
 
 func (pkg *pkgContext) genInputTypes(w io.Writer, t *schema.ObjectType, details *typeDetails) error {
-	contract.Assert(t.IsInputShape())
+	contract.Assertf(t.IsInputShape(), "Object type must be in input shape")
```

Comment:
```suggestion
	contract.Assertf(t.IsInputShape(), "Object type must have input shape")
```
I think that phrases better?